.github/dependency_tests.yaml

```diff
@@ -0,0 +1,12 @@
+broker[docker]: "tests/foreman/ -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -k 'test_positive_get_links'"
+dynaconf[vault]: "tests/foreman/api/test_computeresource_gce.py -k 'test_positive_crud_gce_cr'"
```

Comment:
I'd rather us not use a cloud provider test unless absolutely necessary. Please select another

Reply:
@jyejare Do you have any thoughts about this?

Reply:
updated!